### DISCUSSION TITLE
osu lazer is not responding to my tablet

### DISCUSSION THREAD

**Discussion by Mark-oh1** [ppy/osu#19967] [Q&A]
### Type

Game behaviour

### Bug description

In osu laser settings it is saying that my tablet is not being detected 
I've tried following all the FAQ that is linked in the settings page with no success 
I have also tried reinstalling lazer and my tablet driver and still nothing worked 

You can very barely see that my tablet is putting in inputs but it is all just off the right side of the screen
(By this I mean I can open the notification tab and interact with things using my tablet, but I can no see the censor because it is off the screen )
In the video attached I start by using my mouse, then put my tablet pen on my tablet ad start moving my cursor around and clicking

### Screenshots or videos

https://user-images.githubusercontent.com/111609467/185703964-cc8705f9-20db-4c55-812c-80fc1106fa85.mp4

### Version

2022.816.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/9385345/database.log)
[network.log](https://github.com/ppy/osu/files/9385346/network.log)
[performance.log](https://github.com/ppy/osu/files/9385347/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9385348/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9385349/updater.log)

**Comment by nanashi-1** [comment]
What driver do you use?

**Reply by Mark-oh1** [reply]
Like graphic driver or tablet driver ?

**Reply by nanashi-1** [reply]
I mean tablet driver.

**Reply by Mark-oh1** [reply]
this one
![image](https://user-images.githubusercontent.com/111609467/186880525-d256a4d2-a655-4e35-a05d-3d2d6a9cc603.png)

**Reply by nanashi-1** [reply]
This happened to me too. I had to install [OpenTabletDriver](https://github.com/OpenTabletDriver/OpenTabletDriverl) and uninstall the official drivers. I don't know why this happens but it worked for me.

**Reply by Mark-oh1** [reply]
ok ill give it a try thank you

**Reply by Mark-oh1** [reply]
Hi sorry for comming back late but it dident seem to work
